docker/docker-compose-amd64-cpu.yaml

```diff
@@ -9,20 +9,20 @@ services:
 
   hydrus:
     build:
-      context: ../
-      dockerfile: docker/amd64/hydrus.Dockerfile
-
+      context: ../../hydrus-software-stack
+      dockerfile: docker/amd64/cpu/hydrus.Dockerfile
     privileged: true
     stdin_open: true
     tty: true
     volumes:
-      - ".:/catkin_ws/src"
+      - "../:/home/catkin_ws/src"
```

Comment:
This is a nitpick but is there a reason to move the catkin_ws folder to home?

Reply:
Yes! The real reason is that the entire repository is being copied into the `/catkin_ws/src` directory.
(The last commands in the `amd64/cpu/hydrus.Dockerfile` show this.) Additionally, Docker Compose is mounting the volume from the host computer. As a result, there are effectively two `catkin_ws` directories inside the container:  
`/catkin_ws` (the copy inside the container)  
 `/home/catkin_ws` (the one from the Docker volume mounted from the host)

However, everything is using the `/catkin_ws` directory because it is intended to be the main one. I believe we should always copy the repository into the container to ensure that, even if changes are made on the host machine, the Docker container remains unaffected. This helps to mitigate potential errors during the process.

Reply:
Ig that explains, I understand why. In that case, an environment variable that when set changes the path from /catkin_ws to /home/catkin_ws would be neat. That way, when we are testing code, the script (the ros-entrypoint.sh one) still works and we don't have to type stuff manually, saving a bit of development time.

Reply:
Done. Now it has an extra enviroment variable that we can change when using the argument `./run_docker.sh --volume true` to use the volume or the copy.